src/main/kotlin/com/autonomousapps/transform/StandardTransform.kt

```diff
@@ -64,6 +69,14 @@ internal class StandardTransform(
     androidTestUsages = if (isMainVisibleDownstream) mutableSetOf() else reduceUsages(androidTestUsages)
     computeAdvice(advice, androidTestUsages, androidTestDeclarations, androidTestUsages.size == 1)
 
+
+    /*
+     * Custom JVM source sets like 'testFixtures', 'integrationTest' or other custom source sets and feature variants
+     */
+
+    customJvmUsage = reduceUsages(customJvmUsage)
+    computeAdvice(advice, customJvmUsage, customJvmDeclarations, customJvmUsage.size == 1)
```

Comment:
If you were to change its value, I would expect that some tests in `StandardTransformTest` would fail.

Reply:
Thanks for the pointer. If I understand correctly, this is related to the fact the the Android variants inherit from each other (e.g. the relationship between `implementation`, `releaseImplementation`, `debugImplementation`)

Then this should not concern the _standard_ JVM case we care about here.

In any case I noticed that there is inconsistency between "main usages" and the other usages types.

In "main usages" the `singleVariant` is computed for the original list, while for the "test usages" it is computed on the reduced list. I assume this might actually be wrong the test cases in Android (?).

I added a commit on top in the PR to make this consistent - 96cc3fe24812d465d78d5787d2e56f95e2919d2f. We can remove that gain if it is wrong (or move it to a separate PR).

Reply:
Ok that change fails tests related to "test" dependencies - https://scans.gradle.com/s/4irf2zne7vuhw/tests/overview?outcome=failed

In this case, I leave it as it is.

Reply:
Yeah, we can discuss that in a separate PR if you want. That whole very messy algorithm was developed with TDD. Every decision was made in order to get a test to pass.